types/karma/index.d.ts

```diff
@@ -14,7 +14,10 @@ import Promise = require('bluebird');
 import https = require('https');
 import { Appender } from 'log4js';
 import { EventEmitter } from 'events';
+import constants = require('./lib/constants');
 
+export { constants };
+export const VERSION: typeof constants.VERSION;
```

Comment:
maybe `export {VERSION}` if you just import it normally at the top

Reply:
done, amended